tests/by-name-numprefix/pkgs/by-name/_1/10foo/package.nix

```diff
@@ -0,0 +1,4 @@
+{ someDrv }:
+SomeDrv
+# If we caused an actual Nix failure
+# builtins.trace "This should be on stderr!" throw "This is an error!"
```

Comment:
Can you delete this commented out code I think left over from the [by-name-failure test](https://github.com/Ben-PH/nixpkgs-vet/blob/107_by-name-num-prefix/tests/by-name-failure/pkgs/by-name/fo/foo/package.nix#L3)?